messages/auth.md

```diff
@@ -24,6 +24,14 @@ Invalid request method: %s
 
 Invalid request uri: %s
 
+# warn.invalidClientId
+
+Code Builder web authentication is only supported with the Code Builder connected app. Ignoring client ID: %s
```

Comment:
note: this is not an error message seen by customers.  It's a warning message sent to a debug log.